internal/os/os_test.go

```diff
@@ -0,0 +1,92 @@
+package os
+
+import (
+	"bytes"
+	"strings"
+	"testing"
+
+	root "github.com/equinix/metal-cli/internal/cli"
+	outputPkg "github.com/equinix/metal-cli/internal/outputs"
+	"github.com/spf13/cobra"
+)
+
+func TestCli_RegisterCommands(t *testing.T) {
+	subCommand := "get"
```

Comment:
The `subCommand` should be `"operating-systems"` or `"os"`, and then a test for the `os get` subcommand would call `root.SetArgs(subCommand, "get")`.